Sources/Graphiti/Connection/Edge.swift

```diff
@@ -1,10 +1,10 @@
-protocol Edgeable {
+public protocol Edgeable {
```

Comment:
I'm actually not sure that this protocol needs to be `public` - I think most additions will work simply through `extension Edge`.

```suggestion
protocol Edgeable {
```